### PR TITLE
Azure file copy unique Vm

### PR DESCRIPTION
**Pull request by rokmicrosoft**

### CODE REVIEW

File: Tasks/AzureFileCopy/Utility.ps1 [hunk 1]
```diff
@@ -992,7 +1012,7 @@ function Copy-FilesSequentiallyToAzureVMs
 
         Write-ResponseLogs -operationName 'AzureFileCopy' -fqdn $resourceName -deploymentResponse $copyResponse
         Write-Output (Get-VstsLocString -Key "AFC_CopyCompleted" -ArgumentList $resourceName, $status)
-
+        Publish-Azure-Telemetry  -deploymentResponse $copyResponse -jobId $jobId
```
Comment (chshrikh) [inline Tasks/AzureFileCopy/Utility.ps1]: additional space after Publish-Azure-Telemetry

File: Tasks/AzureFileCopy/Utility.ps1 [hunk 2]
```diff
@@ -2,12 +2,32 @@
```
Comment (chshrikh) [inline Tasks/AzureFileCopy/Utility.ps1]: hold this PR as well, till you publish updated DeploymentUtilities.zip and then need to update the task version

File: Tasks/AzureFileCopy/Utility.ps1 (hunk 1 above)
Comment (rokmicrosoft) [reply Tasks/AzureFileCopy/Utility.ps1]: Removed extra space.

File: Tasks/AzureFileCopy/Utility.ps1 (hunk 2 above)
Comment (rokmicrosoft) [reply Tasks/AzureFileCopy/Utility.ps1]: Updated DeploymentUtilities.zip url and patch version

File: Tasks/AzureFileCopy/task.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 106
+        "Patch": 107
```
Comment (chshrikh) [inline Tasks/AzureFileCopy/task.json]: update this with loc json

Comment (rokmicrosoft) [reply Tasks/AzureFileCopy/task.json]: Done.

**Pull request merged**